default/python/AI/ResourcesAI.py

```diff
@@ -836,7 +836,6 @@ def set_planet_industry_research_influence_foci(focus_manager, priority_ratio):
             # xy = y output when focus x, p for production(INDUSTRY), c for current
             pp, pr, pi = pinfo.possible_output[INDUSTRY]
             rp, rr, ri = pinfo.possible_output[RESEARCH]
-            ip, ir, ii = pinfo.possible_output[INFLUENCE]
```

Comment:
I am in the process of re-writing ResourceAI, this line will be replaced anyway.

Reply:
So let's remove it now.

Reply:
It'll a merge conflict that I have to resolve, but if it is so important for you to get it out, I can do that. I am quite busy this weekend and also the next two, so it will take some time to finish the rewriting anyway.